WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -156,16 +166,24 @@ class MediaCoordinator: NSObject {
     /// Retry the upload of a media object that previously has failed.
     ///
     /// - Parameter media: the media object to retry the upload
+    /// - Parameter isAutomaticRetry: indicates whether an upload is a manual action taken by the user
+    /// or an automagical one done via the `Uploader` protocol.
     ///
-    func retryMedia(_ media: Media, analyticsInfo: MediaAnalyticsInfo? = nil) {
+    func retryMedia(_ media: Media, analyticsInfo: MediaAnalyticsInfo? = nil, isAutomaticRetry: Bool = false) {
```

Comment:
Nitpick, but could possibly shrink this to 'isAutomatic`

Reply:
Agree on this change.